day_6/exercises/burrito.rb

```diff
@@ -13,7 +13,27 @@ def initialize(protein, base, toppings)
   end
 end
 
+def add_topping(topping)
```

Comment:
@JBelta these three methods are a good start. But I'd expect them to be methods defined within the Burrito class. Also, how could you utilize the methods to update the instance variables of a given burrito object?